tools/webpack

```diff
@@ -5,6 +5,7 @@ import os
 import json
 import subprocess
 
+# `NoReturn` wasn't introduced until python 3.5.4
```

Comment:
Drop this commit; as you can see I merged it at a better spot.

Reply:
Oops, meant to do that along with provision's. Done :)